### DISCUSSION TITLE
Alternate Style to Matrix Blocks

### DISCUSSION THREAD

**Discussion by rogertinch** [craftcms/cms#8602] [Ideas]
### Description
The current look of Matrix blocks feels very heavy. Would love if there was an option to give it an alternate look that matched the Variants block style on Craft Commerce. It's more compact, lighter and easier to scan.

**Variants block in Craft Commerce**
![screen shot 2018-10-26 at 1 14 58 pm](https://user-images.githubusercontent.com/11000310/47584899-83039b80-d921-11e8-97ee-d17d48acac5e.png)

**Matrix block in Craft**
![screen shot 2018-10-26 at 1 21 03 pm](https://user-images.githubusercontent.com/11000310/47585178-294fa100-d922-11e8-8203-6eff8915b586.png)

**Comment by brandonkelly** [comment]
Good idea!

**Comment by brandonkelly** [comment]
Worth mentioning here that fields nested within Matrix fields will be able to be placed side-by-side in Craft 3.5 (see https://github.com/craftcms/cms/issues/6346#issuecomment-658919451).
